### DISCUSSION TITLE
Account verification

### DISCUSSION THREAD

**Discussion by glyn** [learn-awesome/learn#260] [Q&A]
I just signed up to LearnAwesome.org and received an account verification email. When I clicked the link, an error was displayed with the message "Your email address could not be verified". The original email contained the statement:

> If you are having any issues with your account, please don't hesitate to contact us by replying to this mail. 

So I replied to the email and got address not found: "Your message wasn't delivered to no-reply@learnawesome.org because the address couldn't be found or is unable to receive email."

Any suggestions for how to proceed?

**Comment by nileshtrivedi** [comment]
Thanks for reporting this issue, @glyn.

You didn't use any social-login, and instead selected email + password combo in the Auth0 screen, right?

**Comment by glyn** [comment]
Correct.

**Reply by nileshtrivedi** [reply]
Hi @glyn 

Can you please test signup with email + password now? I needed to deploy a workaround related to a long-standing limitation with Omniauth's Auth0 integraton. I have tested it myself on the live site and it seems to work. But it will be great if you can verify it.

**Reply by glyn** [reply]
I signed up with a different email address and got the same error message when clicking on the link in the account verification email.

Also, the account verification email from "no-reply@..." still contained the words "If you are having any issues with your account, please don't hesitate to contact us by replying to this mail."

**Reply by glyn** [reply]
Also, is there a way to delete my account as I really don't want two.

**Reply by nileshtrivedi** [reply]
I deleted your second account just now from Auth0's dashboard. Will test verification email flow.

**Reply by nileshtrivedi** [reply]
Although, when I tested yesterday, verification was optional and out-of-band.

**Reply by nileshtrivedi** [reply]
So, this email was sent by Auth0 and not our webapp. I have updated the contact email address to team@learnawesome.org instead of the no-reply one.
